blacklight.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.7'
 
-  s.add_dependency "rails", '>= 7.0', '< 8'
+  s.add_dependency "rails", '>= 7.0', '< 9'
```

Comment:
Pretty sure you don't actually have to change this to test on a beta, to bundler `8.0.0.beta1` is < `8`. 

As long as we don't release... are we concerned about accidentally doing a release that allows any 8.x before 8.0.0 final is out and we're sure it's good, if we end up doing a release in the interim?

Reply:
Main branch is 9.0.0.alpha.  I'm pretty sure we're not going to release from main before the final is out.